{
    "command_failed": "Příkaz selhal!"
}